File: Pard.Angular/Pard-Application/src/app/records/models/record.ts
import { Location } from './location';

export class Record {
    id: string;
    title: string;
    description: string;
    isDone: boolean;
    addedAt: Date;
    finishedAt?: Date;
    userId: string;
    location?: Location;

    constructor(title: string, description: string, isDone: boolean) {
        this.title = title;
        this.description = description;
        this.isDone = isDone;
    }
}

File: Pard.Domain/Entities/Records/Record.cs
using System;
using Pard.Domain.Entities.Locations;

namespace Pard.Domain.Entities.Records
{
    public class Record
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public bool IsDone { get; set; }
        public DateTime AddedAt { get; set; }
        public DateTime? FinishedAt { get; set; }
        public Guid UserId { get; set; }
        public bool IsDeleted { get; set; }
        
        public Location Location { get; set; }
    }
}
